apps/auth0-rules/src/connect-user.ts

```diff
@@ -1,7 +1,13 @@
 import got from 'got';
-import { handleError } from './handle-error';
 import type { Rule } from './types';
 
+const handleError = (err: unknown): Error => {
+  if (err instanceof Error) {
+    return err;
+  }
+  return new Error('Unexpected Error');
+};
+
```

Comment:
Why can't we use the import?

Reply:
cause auth0 does allow us to do that and we dont have a setup for bundling all into one file - we could do it now but we have a ticket to revamp the rules at some point so